src/Codata/Musical/Colist/Infinite-merge.agda

```diff
@@ -195,7 +195,7 @@ module _ {a p} {A : Set a} {P : A → Set p} where
       ... | inj₂ q | P.refl | q≤p =
         Prod.map there
                  (P.cong (there ∘ _⟨$⟩_ (Inverse.from (Any-⋎P xs)) ∘ inj₂))
-                 (rec (♭ xss , q) (s≤′s q≤p))
+                 (rec {♭ xss , q} (s≤′s q≤p))
```

Comment:
Amazingly, this one is eliminable as well, but I have left it in as a comment, for the sake of documentation in the proof.